nginx.conf

```diff
@@ -0,0 +1,20 @@
+server {
+  listen 80;
+  sendfile on;
+  default_type application/octet-stream;
+
+  gzip on;
```

Comment:
We might consider pre-gzipping the bundle files during build process. So that the runtime load on nginx is smaller.
This can be done using [`gzip_static  on;`](http://nginx.org/en/docs/http/ngx_http_gzip_static_module.html)